File: 8kyu_Barking_mad.rb
# https://www.codewars.com/kata/barking-mad/ruby

class Dog
  def initialize(breed)
    @breed=breed
  end
end

snoopy=Dog.new("Beagle")

def snoopy.bark()
  "Woof"
end

scoobydoo=Dog.new("Great Dane")

def scoobydoo.bark()
  "Woof"
end

File: 8kyu_Barking_mad.js
// Challenge: https://www.codewars.com/kata/54dba07f03e88a4cec000caf

function Dog (breed) {
  this.breed = breed;
  this.bark = function() {
  return "Woof";
};
}

var snoopy = new Dog("Beagle");



var scoobydoo = new Dog("Great Dane");
